ethereum/core/src/main/java/org/hyperledger/besu/ethereum/storage/keyvalue/KeyValueSegmentIdentifier.java

```diff
@@ -35,7 +35,8 @@ public enum KeyValueSegmentIdentifier implements SegmentIdentifier {
   BACKWARD_SYNC_BLOCKS(new byte[] {14}),
   BACKWARD_SYNC_CHAIN(new byte[] {15}),
   SNAPSYNC_MISSING_ACCOUNT_RANGE(new byte[] {16}),
-  SNAPSYNC_ACCOUNT_TO_FIX(new byte[] {17});
+  SNAPSYNC_ACCOUNT_TO_FIX(new byte[] {17}),
+  CHAIN_PRUNER_STATE(new byte[] {18});
```

Comment:
will this make database backward incompatible?

Reply:
Yea, I think so. I'm just adding a new segment so it should be backward-compatible.

Reply:
Did manual testing earlier, besu with this new column has no problem mounting to a database created by the old besu.